src/contribute/docker-images.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: default
+title: 'Beam Docker Images'
+permalink: /contribute/docker-images/
+---
+
+# Docker Images
+
+You can bootstrap or reproduce an Apache Beam developer environment by using
+our provided [Docker scripts](https://github.com/apache/beam/tree/master/sdks/java/build-tools/src/main/resources/docker).
+In this directory you will find scripts to build and run docker images for
+different purposes:
+
+- [file](#file): Create a Docker container from a Beam source code .zip file
+  in a given environment. It is useful to test/validate the release votes
```

Comment:
I'm not sure if this phrase should be "for release votes" (rather than "the release votes") or maybe it was meant to be "the release notes", which would also make sense. either change would work but not sure which is intended here.